src/components/booking/SkeletonSessionCard.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+import { Skeleton } from "../ui/skeleton";
+
+const SkeletonSessionCard = () => {
+  return (
+    <Skeleton className="rounded-sm border px-6 flex-col gap-1 py-4 min-h-24 flex font-medium bg-secondary opacity-40">
+      <Skeleton
+        className="bg-tertiary opacity-20 h-[24px]"
```

Comment:
```suggestion
        className="bg-tertiary opacity-20 h-6"
```

Reply:
try use tailwind default sizes and stick to a 4px scale

Reply:
Try use tailwind default sizes and stick to a 4px scale

Also h-[18px] can probably be h-4?

Reply:
done